spec/index.bs

```diff
@@ -1292,6 +1291,8 @@ If any of the following conditions are violated:
 
 ## GPUBindGroup ## {#bind-groups}
 
+A {{GPUBindGroup}} defines a set of resources to be bound and how the resources are used in shader stages.
```

Comment:
"to be bound together in a group", perhaps?

Reply:
done.